spec/models/performance_data/statistics_spec.rb

```diff
@@ -0,0 +1,135 @@
+require 'rails_helper'
+require 'performance_data/statistics'
+require 'performance_data/test_helpers/statistics'
+require 'gds_api/test_helpers/content_store'
+
+module PerformanceData
+  describe Statistics do
+    include PerformanceData::TestHelpers::Statistics
+    include GdsApi::TestHelpers::ContentStore
+
+    before do
+      random_need = GovukSchemas::RandomExample.for_schema(
+        frontend_schema: "need"
+      )
+
+      apply_for_a_uk_visa_need = random_need.merge_and_validate(
+        "details" => {
+            "role" => "As a non-EEA national",
+            "goal" => "I need to apply for a UK visa",
+            "benefit" => "So that I can come to the UK to visit, study or work",
+            "justifications" => ["It's something only government does"],
+            "met_when" => [
+                "Finds out how whether they're eligible",
+                "How to apply",
+                "What documents to provide"
+            ],
+        }
+      )
+
+      random_content = GovukSchemas::RandomExample.for_schema(
+        frontend_schema: "specialist_document"
+      )
+
+      supertype_fields = apply_for_a_uk_visa_need.keys.select { |field| field.end_with? "supertype" }
+      fields_to_exclude = %w(
+        rendering_app
+        withdrawn_notice
+        last_edited_at
+        updated_at
+        first_published_at
+        publishing_app
+        need_ids
+        format
+        phase
+        publishing_request_id
+      ) + supertype_fields
+
+      @apply_uk_visa_content = random_content.merge_and_validate(
+        title: "Apply for a UK visa",
+        links: {
+            meets_user_needs: [apply_for_a_uk_visa_need.except(*fields_to_exclude)]
+        }
+      )
+
+      @apply_uk_visa_content_multipart = GovukSchemas::RandomExample.for_schema(
+        frontend_schema: "guide"
+      ).merge_and_validate(
+        title: "Apply for a UK visa 1",
+        links: {
+            meets_user_needs: [apply_for_a_uk_visa_need.except(*fields_to_exclude)]
+        },
+        "details" => {
+            "parts" => [
+                {
+                    "slug" => 'part-1',
+                    "title" => 'Part 1',
+                    "body" => 'Part 1',
+                },
+                {
+                    "slug" => 'part-2',
+                    "title" => 'Part 2',
+                    "body" => 'Part 2',
+                },
+                {
+                    "slug" => 'part-3',
+                    "title" => 'Part 3',
+                    "body" => 'Part 3',
+                },
+            ]
+        },
+      )
+
+      @slug = 'apply-uk-visa'
+    end
+
+    context "problem_reports" do
```

Comment:
The name of this context makes me think the specs inside it are just about the `problem_reports` method, but we're actually testing all the methods in each test.  To make it clearer we could either drop this context, or split up the tests so we have separate ones for each method. No strong preference either way.